cfme/tests/webui/test_expression_editor.py

```diff
@@ -0,0 +1,44 @@
+import fauxfactory
+import pytest
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.blockers import BZ
+
+
+class TestDeleteExpressions(object):
+    filter_value = 'beautifulpotato'
+    filt = "Cloud Volume : Name"
+    collection = "volumes"  # any collection with advanced search
+    destination = "All"
+
+    @pytest.mark.meta(automates=[BZ(1720216)])
+    @pytest.mark.parametrize("operator", ["not", "and", "or"])
+    # remove NOT is 5.11 feature, see BZ(1720216)
+    @pytest.mark.uncollectif(lambda operator, appliance:
+                             operator == "not" and appliance.version < "5.11")
+    def test_delete_operator(self, appliance, operator):
+        """
+        Polarion:
+            assignee: anikifor
+            casecomponent: WebUI
+            caseimportance: medium
+            initialEstimate: 1/30h
+        """
+        view = navigate_to(
+            getattr(
+                appliance.collections,
+                self.collection),
+            self.destination)
+        name = fauxfactory.gen_alphanumeric()
+        view.entities.search.save_filter(
+            f'fill_field({self.filt}, =, {self.filter_value});'
```

Comment:
since this's one big string. I'd advice you use doc blocks like """ blabla """ instead.

Reply:
nice, didn't know that this is possible with regular strings, thanks for advice